DataFormats/TrackSoA/interface/TracksHost.h

```diff
@@ -1,47 +1,11 @@
 #ifndef DataFormats_Track_TracksHost_H
 #define DataFormats_Track_TracksHost_H
```

Comment:
Could you change these to
```suggestion
#ifndef DataFormats_TrackSoA_interface_TracksHost_H
#define DataFormats_TrackSoA_interface_TracksHost_H
```
?